### PR TITLE
Social Networks List

### PR DESCRIPTION
**Pull request by jonthomp**
Adds a `SocialNetowrksList` for displaying a `List` of social media icons, also adds `bio` prop to Profile. Updates `Profile` and `MediaBodySocial` to use `SocialNetworksList`

### CODE REVIEW

File: src/components/SocialNetworksList/SocialNetworksList.react.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +asButtons?: boolean,
+  +prefix?: string,
+  +itemsObjects?: Array<itemObject>,
+  +items?: Array<React.Node>,
+|};
+
+function listItemFromObjectFactory(asButtons, iconPrefix) {
```
Comment (AaronCoplan) [inline src/components/SocialNetworksList/SocialNetworksList.react.js]: Here we should type the two arguments as well as the return type for this function

File: src/components/SocialNetworksList/SocialNetworksList.react.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+function SocialNetworksList(props: Props): React.Node {
+  const { children, className, asButtons, prefix = "fe" } = props;
+  const classes = cn("social-links", className);
+
+  const getObjectListItem = listItemFromObjectFactory(asButtons, prefix);
+
+  const items =
+    (props.itemsObjects && props.itemsObjects.map(getObjectListItem)) ||
```
Comment (AaronCoplan) [inline src/components/SocialNetworksList/SocialNetworksList.react.js]: Let's destructure itemsObjects and items on line 40

File: src/components/Avatar/Avatar.react.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { Icon } from "../";
 import cn from "classnames";
 import AvatarList from "./AvatarList.react";
 
-type Props = {|
+export type Props = {|
```
Comment (AaronCoplan) [inline src/components/Avatar/Avatar.react.js]: I can't seem to find where we are importing this

Comment (jonthomp) [reply src/components/Avatar/Avatar.react.js]: It's not actually used in this PR, slipped in from some other stuff I was working on

**Comment by AaronCoplan** [comment]
Will approve once build failure is resolved!  Looking good!

**Pull request merged**